docs/develop/python/temporal-clients.mdx

```diff
@@ -139,7 +139,38 @@ if __name__ == "__main__":
 
 ## Connect to Temporal Cloud {#connect-to-temporal-cloud}
 
-**How to connect to Temporal Cloud using the Python SDK**
+### How to connect to Temporal Cloud using an API key {#connect-to-temporal-cloud-api-key}
+
+To use an [API key](/cloud/api-keys) with the Temporal Python SDK, you will need to gather some information first:
```

Comment:
See complaint about "gather some information" phrasing wrt the Go SDK (applies to all of them, won't comment in every spot)